pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1450,7 +1453,13 @@ func calculateRequestedVCPUs(cpuTopology *CPUTopology) uint32 {
 
 func CalculateNetworkQueueNumberAndGetCPUTopology(vmi *v1.VirtualMachineInstance) (uint32, *CPUTopology) {
 	cpuTopology := getCPUTopology(vmi)
-	return calculateRequestedVCPUs(cpuTopology), cpuTopology
+	queueNumber := calculateRequestedVCPUs(cpuTopology)
+
+	if queueNumber > multiQueueMaxQueues {
+		log.Log.Infof(fmt.Sprintf("Capped the number of queues to be the current maximum of tap device queues: %d", multiQueueMaxQueues))
```

Comment:
Can we give this an increase verbosity level? Maybe `3`? Otherwise it will be printed on every conversion (on every sync between virt-handler and virt-launcher).

Reply:
Done.